benchmarks/benchmark/tools/dlio/modules/parallelstore_storage/ps_pv.tpl

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: ${pv_name}
+spec:
+  storageClassName: ""
+  capacity:
+    storage: 12Ti
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
This might be "Delete"?

Reply:
for static provision we don't want delete.